### PR TITLE
@eessex => Gucci metadata

### PR DESCRIPTION
**Pull request by kanaabe**
Addresses points in: https://waffle.io/artsy/publishing/cards/5a032dec1b23d70089e701fd

### CODE REVIEW

**Comment by eessex** [comment]
This would cover individual videos, I wonder if we want info for the series at large as well?

**Comment by kanaabe** [comment]
Yess good idea. I'll add that in

File: client/apps/settings/client/curations/gucci/test/section.test.js [hunk 1]
```diff
@@ -6,40 +6,64 @@ import ImageUpload from 'client/apps/edit/components/admin/components/image_uplo
...
+    thumbnail_image: 'http://thumbnail.jpg',
+    email_image: 'http://emailimage.jpg',
+    social_image: 'http://socialimage.jpg'
   }
 
+  beforeEach(() => {
+    props = {
+      section,
+      onChange: jest.fn()
+    }
```
Comment (eessex) [inline client/apps/settings/client/curations/gucci/test/section.test.js]: Good call on putting the mock here

**Comment by eessex** [comment]
Looks great, thanks!

**Pull request merged**